ops/check-changed/main.py

```diff
@@ -13,9 +14,12 @@
     r'ops/check-changed/.*',
     r'^go\.mod',
     r'^go\.sum',
-    r'^pnpm-lock\.yaml',
     r'ops/check-changed/.*'
 ]
+with open("../../nx.json") as file:
+    nx_json_data = json.load(file)
+for file in nx_json_data["implicitDependencies"].keys():
+    REBUILD_ALL_PATTERNS.append(file)
```

Comment:
nx.json https://github.com/ethereum-optimism/optimism/blob/develop/nx.json#L6

looks like this:

```json
{
  "implicitDependencies": {
    "nx.json": "*",
    "tsconfig.json": "*",
    ".foundryrc": "*",
    ".nvmrc": "*"
  }
}
```